analysis2_manuscript.tex

```diff
@@ -896,6 +896,11 @@ \section*{Methods}
     in which chromosomes were simulated separately from one another.
     All simulations were executed using the \slim engine with a burn-in of $10N$ generations
     (where $N$ is the initial population size), and a scaling factor of 1 (no scaling).
+    We note in passing that aggressive scaling of forward-in-time simulations
+    can lead to quite different outcomes when selection in involved \cite{dabi2025population},
```

Comment:
Should also cite Uricchio and Hernandez and Ferrari et al here (https://doi.org/10.1534/genetics.113.156935 and https://doi.org/10.1101/2024.04.27.591463).

Also, maybe we just note this instead of noting it "in passing"?